app/helpers/application_helper.rb

```diff
@@ -318,4 +317,33 @@ def maybe_css_class
     CSS_CLASS_MAYBE
   end
 
+  # public method for accessing the CSS class for the main title of a page
+  def content_title_css_class
+    CSS_CONTENT_TITLE_CLASS
+  end
+
+  # public method to render the main title of a page
+  def content_title(title, user: nil, id: nil, classes: [])
+    tag.h1 title,
+           class: classes + with_admin_css_class_if_needed(user, [content_title_css_class]),
+           id: id
+  end
+
+  def user_name_for_display(user)
+    user_name = user&.full_name
```

Comment:
I think a guard clause - rather than repeated checks on `user` - would make sense here (https://github.com/rubocop-hq/ruby-style-guide#nested-conditionals).  So, something like:
```
return '' unless user
```
at the top of the method.

Reply:
> I think a guard clause - rather than repeated checks on `user` - would make sense here (https://github.com/rubocop-hq/ruby-style-guide#nested-conditionals). So, something like:
> 
> ```
> return '' unless user
> ```
> 
> at the top of the method.

I like how more compact it looks this way, but it might make others stop and wonder about if I'm actually checking for that, as they could not notice the `&.`. I'll change it.